app/helpers/accounts_helper.rb

```diff
@@ -104,4 +104,26 @@ def brief_account_info(contact)
         end
     text.html_safe
   end
+
+  # We have too much login in the account views
+  # - a helper that abstracts the logic to the backend
+  def display_pipeline(pipeline)
```

Comment:
These might be better merged into a single function, as they repeat the same logic a few times.x

Reply:
pipeline, won, and lost are all one function now